java/query/query-core/src/main/java/sleeper/query/core/rowretrieval/LeafPartitionQueryExecutor.java

```diff
@@ -76,12 +76,13 @@ public CloseableIterator<Row> getRows(LeafPartitionQuery leafPartitionQuery) thr
         Schema tableSchema = tableProperties.getSchema();
         String compactionIteratorClassName = tableProperties.get(TableProperty.ITERATOR_CLASS_NAME);
         String compactionIteratorConfig = tableProperties.get(TableProperty.ITERATOR_CONFIG);
-        ConfigStringIterator compactionIterator;
-        ConfigStringIterator queryIterator;
+        String compactionFilters = tableProperties.get(TableProperty.FILTERS_CONFIG);
```

Comment:
It doesn't look like this is tested. That can go in QueryExecutorTest.ApplyIterators.